docs/next/guides/building-and-testing/modules.md

```diff
@@ -481,352 +275,822 @@ new Handsontable(container, {});
 
 ```js
 import Handsontable from 'handsontable/base';
+import 'handsontable/dist/handsontable.full.css';
 import { registerLanguageDictionary, plPL } from 'handsontable/i18n';
 
-registerLanguageDictionary(plPL);
+registerLanguageDictionary(plPL.languageCode, plPL);
 
-// use the locales
+// set the `language` configuration option
 new Handsontable(container, {
   language: 'pl-PL',
-// rest of the settings
 });
 ```
 
-And that is all! You can use the PL-pl translation!
+:::tip
+#### Optimizing Moment.js locales
 
-### Optimizing moment.js locales
+By default, [Moment.js](https://momentjs.com/) (Handsontable's dependency) comes with all possible locales, which increases the bundle size.
 
-If you want to decrease the bundle size even more you can also focus on optimizing moment.js locales. There are different methods of doing so but this guide focuses on using the webpack's [IgnorePlugin](https://webpack.js.org/plugins/ignore-plugin/). For another option, you can check [this tutorial](https://github.com/jmblog/how-to-optimize-momentjs-with-webpack) directly.
-
-Moment.js can be heavyweight because when you type `var moment = require('moment')` in the code you get all locales as default. To be more selective with your choice you can first, use the `IgnorePlugin` that removes all locales:
+To [optimize Moment.js locales](https://github.com/jmblog/how-to-optimize-momentjs-with-webpack), use [webpack's `IgnorePlugin`](https://webpack.js.org/plugins/ignore-plugin/):
 
 ```js
 const webpack = require('webpack');
 module.exports = {
   //...
   plugins: [
-    // Ignore all locale files of moment.js
+    // ignore all Moment.js locale files
     new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
   ],
 };
 ```
 
-And then explicitly load selected locales:
+And then explicitly load Moment.js, importing only those locales that you need:
 
 ```js
-import moment from 'moment';
-import 'moment/locale/ja';
 import Handsontable from 'handsontable/base';
+import 'handsontable/dist/handsontable.full.css';
 import { registerCellType, DateCellType } from 'handsontable/cellTypes';
 
+// explicitly import Moment.js
+import moment from 'moment';
+// explicitly import a Moment.js locale of your choice
+import 'moment/locale/ja';
+
+// register the Moment.js locale of your choice
 moment.locale('ja');
 registerCellType(DateCellType);
 
 new Handsontable(container, {
   type: 'date',
 });
 ```
+:::
 
-## Using modules with frameworks
+### Using modules with framework wrappers
 
-You can also use modules with Handsontable's framework wrappers:
+You can use modules with Handsontable's framework wrappers:
 - [Using modules with React](@/guides/integrate-with-react/react-modules.md)
 - [Using modules with Angular](@/guides/integrate-with-angular/angular-modules.md)
 - [Using modules with Vue 2](@/guides/integrate-with-vue/vue-modules.md)
 - [Using modules with Vue 3](@/guides/integrate-with-vue3/vue3-modules.md)
+
+### Tree shaking
```

Comment:
I don't see the point of this section. I think that this section should be removed.

Do we actually support all of the bundlers listed below? I think that we only test with Webpack and ParcelJS (see:  https://docs.google.com/spreadsheets/d/1MEe4tF2xseu6QI3N6xC4FcIevG_p0q2uWZt0JpURJfk/edit#gid=0)

That said, I think that this page is only relevant for developers who use bundlers. Such information is missing on this page. I think that we should add somewhere close to the top:

> Loading Handsontable by module requires an environment that supports the `import` syntax. We actively test Handsontable using Webpack and ParcelJS bundlers that provide such ability. If the methods presented on this page don't work with your bundler, please report it as a bug. Direct use of ES Modules is not supported at the moment.

@budnix is the above text correct?

Reply:
The Handsontable distribution package is compatible with CommonJS and ES. I tested Handsontable with Webpack {2,3,4,5}, Parcel {1,2}, Rollup {1,2}, Vite and didn't find any issues. I would say that the text is partially correct. We do not test the code within any bundlers. We build a UMD package with Webpack 4 and run the E2E tests on it but that's it.

Reply:
If we **can** present a comprehensive list of all officially supported bundlers, I'd do it in a table, like with [HyperFormula version support](https://handsontable.com/docs/formula-calculation/#hyperformula-version-support).

If we can't (for the lack of regular testing), I'd do it like [this](https://github.com/handsontable/handsontable/pull/9059/commits/36fbba274c0e9e62fc0bab9472babba95beed578#diff-3b9c74dc2c4138891da8b067a2113b8d2aa6bf1b2ff3e1244a4b71923fbec14eR27-R35).

Reply:
@kirszenbaum this proposal is almost good, but I don't agree on saying that we recommend these bundlers. It is not our job to recommend a bundler, because that's a prior decision by the app developer.

I would say this:

```
To use Handsontable modules, your environment has to support the `import` syntax, which is typically provided by a bundler.

We have successfully tested Handsontable with the following bundlers:
- [webpack](https://webpack.js.org)
- [Parcel](https://parceljs.org)
- Rollup
- Vite

If Handsontable modules don't work with your bundler, [report it as a bug](https://github.com/handsontable/handsontable/issues/new?assignees=&labels=&template=01-handsontable.md&title=).
```